### PR TITLE
Tests: Fix a comment

### PR DESCRIPTION
**Pull request by fridokus**
Fix a comment that was false

### CODE REVIEW

File: test/functional/mempool_resurrect.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def run_test(self):
         for node in self.nodes:
             node.invalidateblock(blocks[0])
 
-        # mempool should be empty, all txns confirmed
+        # All txns should be back in mempool with 0 confirmations
```
Comment (MarcoFalke) [inline test/functional/mempool_resurrect.py]: Could remove the comment, since it is mentioned two lines up already?

Comment (fridokus) [reply test/functional/mempool_resurrect.py]: Removed the first occurrence instead, to put the information closer to where it's relevant.

**Comment by MarcoFalke** [comment]
Please keep your commits squashed for trivial changes. See https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#squashing-commits

**Comment by fridokus** [comment]
Squashed the commits

**Comment by fanquake** [comment]
utACK 086fc83

**Pull request merged**